platform/ext/target/nordic_nrf/common/nrfx/mdk/nrf5340_application.h

```diff
@@ -0,0 +1,3462 @@
+/*
```

Comment:
@anangl I've got a question about the vendor headers, e.g. this one:
In TF-M there seems to be a convention for the IRQ enum to be defined in a header called "platform_irq.h"
How is it going to work with this nrfx header?

Reply:
These headers (`platform_irq.h`) are only included by other vendor files, like `tfm_peripherals_def.h`, so IMO we don't need to provide such header, we can just include `nrf.h` instead.

Reply:
Yeah, or have a "dummy" platform_irq.h header that includes nrfx.h.
I'd say that's safer option - those headers might be there for historical reasons